controllers/idler/idler_controller_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/codeready-toolchain/toolchain-common/pkg/cluster"
+
```

Comment:
same here, let's keep the imports grouped

Reply:
done in [777b019](https://github.com/codeready-toolchain/member-operator/pull/357/commits/777b019b9dc2a2da36de0b0e2754b8012b10f7d8) and sorted imports